include/yaclib/fault/chrono.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <yaclib/fault/detail/chrono/steady_clock.hpp>
+#include <yaclib/fault/detail/chrono/system_clock.hpp>
+
+namespace yaclib_std::chrono {
+
+#ifdef YACLIB_FAULT_TEST_ON
```

Comment:
I think we want one macro:
YACLIB_FAULT -- add to library fault injection

If we want to test our library with fault injection we write -DYACLIB_FAULT=ON -DSEED/THREAD=*